test/quantization/test_backward_compatibility.py

```diff
@@ -69,6 +77,51 @@ def remove_prefix(text, prefix):
         self.assertEqual(qmodule_scripted(input_tensor), expected, atol=prec)
         self.assertEqual(qmodule_traced(input_tensor), expected, atol=prec)
 
+    def _test_op_graph(self, qmodule, subname=None, input_size=None, input_quantized=True,
+                       generate=False, prec=None, new_zipfile_serialization=False):
+        r"""
+        Input: a floating point module
+
+        If generate == True, traces and scripts the module and quantizes the results with
+        PTQ, and saves the results.
+
+        If generate == False, traces and scripts the module and quantizes the results with
+        PTQ, and compares to saved results.
+        """
+        input_file, state_dict_file, scripted_module_file, traced_module_file, expected_file = \
+            get_filenames(self, subname)
+
+        # only generate once.
+        if generate and qengine_is_fbgemm():
+            input_tensor = torch.rand(*input_size).float()
+            torch.save(input_tensor, input_file)
+
+            # convert to TorchScript
+            scripted = torch.jit.script(qmodule)
+            traced = torch.jit.trace(qmodule, input_tensor)
+
+            # quantize
+
+            def _eval_fn(model, data):
+                model(data)
+
+            qconfig_dict = {'': torch.quantization.default_qconfig}
+            scripted_q = torch.quantization.quantize_jit(
+                scripted, qconfig_dict, _eval_fn, [input_tensor])
+            traced_q = torch.quantization.quantize_jit(
+                traced, qconfig_dict, _eval_fn, [input_tensor])
+
+            torch.jit.save(scripted_q, scripted_module_file)
+            torch.jit.save(traced_q, traced_module_file)
+            torch.save(scripted_q(input_tensor), expected_file)
+
+        input_tensor = torch.load(input_file)
+        qmodule_scripted = torch.jit.load(scripted_module_file)
+        qmodule_traced = torch.jit.load(traced_module_file)
+        expected = torch.load(expected_file)
+        self.assertEqual(qmodule_scripted(input_tensor), expected, atol=prec)
```

Comment:
Is there any value in comparing graph with eager expected outputs?

Reply:
There is definitely value in that, looks like that's already done in `test_quantize_jit.py`, so by those tests and the fact that both eager and graph are tested here for BC numerics, we can prove that eager vs graph condition is satisfied indirectly.